routing_model/flow.md

```diff
@@ -11,7 +11,7 @@ Notes:
 Todo:
```

Comment:
This file should be removed, it is completly out of date, only the index.html is up to date.